client/blocks/post-status/test/index.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import identity from 'lodash/identity';
+import Gridicon from 'components/gridicon';
+
+describe( 'PostStatus', ( ) => {
+	let PostStatus;
+
+	before( ( ) => {
```

Comment:
>No filler spaces in empty constructs (e.g., `{}`, `[]`, `fn()`).

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#spacing

Will have to see if we can get an ESLint rule to enforce this.

There's several instances of this through the file.